app/utils/browser/index.js

```diff
@@ -20,7 +20,11 @@ if (env.NODE_ENV === 'development') {
 let analytics;
 const gaKey = config.get('integrations.ga.key');
 if (gaKey) {
-  analytics = new Analytics(gaKey);
+  try {
+    analytics = new Analytics(gaKey);
+  } catch (err) {
+    console.error('Google Analytics error:', err);
```

Comment:
Just out of curiosity – are the `console.errors` caught by some tool such as Bugsnag or something?

Reply:
Not in this case unfortunately since this error happens in a node env. We have bugsnag enabled for the renderer process, app context basically.

Reply:
How difficult to catch errors in both processes and this preload script? Also, does electron expose a `onPreloadError` type handler or way to hook into preload script issues?

Reply:
Good question. I'll check